src/style.css

```diff
@@ -0,0 +1,110 @@
+* {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  box-sizing: border-box;
+  padding: 0;
```

Comment:
and don't reset padding for `*` selector. it will be better to some special elements.

Reply:
Ok, I misunderstood. Thanks for the clarification, hope this time it's ok.